bot/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python ./Engine.py "TOKEN" "starting"
```

Comment:
Should we just `CP ../.env .env` and use load_env() rather than *sys.argv[1:]

That way we can also define `MAIN_GUILD_ID` in env which would be the id of your main/testing discord server